controllers/auth-controller.js

```diff
@@ -48,44 +63,86 @@ const signIn = async (req, res) => {
 
   const payload = {
     id: user._id,
-  }
+  };
 
-  const token = jwt.sign(payload, JWT_SECRET, {expiresIn: "1h"});
+  const token = jwt.sign(payload, JWT_SECRET, { expiresIn: "1h" });
   await User.findByIdAndUpdate(user._id, { token });
 
   res.json({
     token,
     user: {
-        email: email,
-        subscription: user.subscription,
-    }
+      email: email,
+      subscription: user.subscription,
+    },
   });
 };
 
-const getCurrent = (req,res) => {
-    const { email, subscription } = req.user;
+const getCurrent = (req, res) => {
+  const { email, subscription } = req.user;
 
-    res.json({        
-        email,
-        subscription,
-    });
-}
+  res.json({
+    email,
+    subscription,
+  });
+};
 
 const logout = async (req, res) => {
-    const { _id } = req.user;
+  const { _id } = req.user;
+
+  await User.findByIdAndUpdate(_id, { token: "" });
+
+  res.status(204).json({
+    status: "success",
+    code: 204,
+    message: "No Contetnt",
+  });
+};
 
-    await User.findByIdAndUpdate(_id, {token: ""});
+const verifyToken = async (req, res) => {
+  const { verificationToken } = req.params;
+  
+  const user = await User.findOne({verificationToken});
+  
+  if (!user) {
+    throw HttpError(404, "User not found.");
+  }
+  
+  await User.findByIdAndUpdate(user._id, {verificationToken: "", verify: true});
```

Comment:
verificationToken should be null

Reply:
fixed

Reply:
have you tested your changes? Will it work correctly?

Reply:
[Photo of test](https://postimg.cc/zHyDb2Jt)